utilities/blob_db/blob_db_impl.cc

```diff
@@ -187,6 +198,26 @@ Status BlobDBImpl::Open(std::vector<ColumnFamilyHandle*>* handles) {
   }
   db_impl_ = static_cast_with_check<DBImpl, DB>(db_->GetRootDB());
 
+  // Initialize SST file <-> oldest blob file mapping if garbage collection
+  // is enabled.
+  if (bdb_options_.enable_garbage_collection) {
+    std::vector<LiveFileMetaData> live_files;
+    db_->GetLiveFilesMetaData(&live_files);
+
+    InitializeParentSstMapping(live_files);
```

Comment:
Maybe it's just me but I feel the word "parent" confusing. To me it's blob and its index. The word "parent" isn't intuitive for me to understand what it means. Can we think about a better way to represent to relationship between the two?

Reply:
How about we simply call these "blob to SST" mapping, or something along those lines?

Reply:
IT sounds good.